microsoft/PowerToys
# Bug report tool

## Summary of the Pull Request

_What is this about?_
Simple standalone tool for bug reporting. It is a win32 console application that takes the path of the resulting zip folder as cmd argument. The resulting zip folder includes PowerToys folder from AppData. 

## PR Checklist
* [X] Applies to #726

## Info on Pull Request

_What does this include?_
- Standalone application `BugReportTool.exe`
- Separate solution `BugReportTool.sln`
- External zip library as git submodule

The tool performs the following steps: 
- Copiy the PowerToys folder from AppData into the Temp folder 
- Replace all sensitive data in json files and delete some files
- Create zip folder on the desktop if cmd argument was not send

## Validation Steps Performed

_How does someone test & validate?_
Buid `BugReportTool.sln` and installer. Then try to install it and verify if the tool is present under `C:\Program Files\PowerToys\BugReportTool` and it works. Try passing the location of the resulting zip folder as a cmd argument.
